Common/include/option_structure.hpp

```diff
@@ -1162,17 +1164,24 @@ enum ENUM_PARAM {
   AIRFOIL = 17,		           /*!< \brief Airfoil definition as design variables. */
   SURFACE_FILE = 18,		     /*!< Nodal coordinates set using a surface file. */
   CUSTOM = 19,                /*!< 'CUSTOM' for use in external python analysis. */
-  CST = 20                /*!< \brief CST method with Kulfan parameters for airfoil deformation. */
+  CST = 20,                /*!< \brief CST method with Kulfan parameters for airfoil deformation. */
+  NO_DEFORMATION = 21,		       /*!< \brief No Deformation. */
+  HTP_INCIDENCE = 22,			         /*!< \brief Incidence of the HTP. */
+  ANGLE_OF_ATTACK = 101,	   /*!< \brief Angle of attack for airfoils. */
+  FFD_ANGLE_OF_ATTACK = 102	 /*!< \brief Angle of attack for FFD problem. */
 };
 static const map<string, ENUM_PARAM> Param_Map = CCreateMap<string, ENUM_PARAM>
 ("FFD_SETTING", FFD_SETTING)
 ("FFD_CONTROL_POINT_2D", FFD_CONTROL_POINT_2D)
+("FFD_ANGLE_OF_ATTACK", FFD_ANGLE_OF_ATTACK)
 ("FFD_CAMBER_2D", FFD_CAMBER_2D)
 ("FFD_THICKNESS_2D", FFD_THICKNESS_2D)
 ("HICKS_HENNE", HICKS_HENNE)
+("ANGLE_OF_ATTACK", ANGLE_OF_ATTACK)
```

Comment:
Yep! it is working great, I'll add an example. To compute the correct gradient (at cte. CL), the critical number is dCD/dCL, you can provide that number or SU2 will be able to compute that during runtime. There are some adjustments of the python files and the angle of attack is stored on the restart and geometry files. It is a small implementation with a lot of details. By the way, the SU2 screen output doesn't show the AOA gradient... I should modify that.